docs/src/fts.md

```diff
@@ -35,42 +35,83 @@ table = db.create_table(
 
 The FTS index must be created before you can search via keywords.
 
-```python
-table.create_fts_index("text")
-```
+=== "> 0.11.0"
+
+    ```python
+    table.create_scalar_index("text", "INVERTED")
+    ```
+
+=== "<= 0.11.0"
+
+    ```python
+    table.create_fts_index("text")
+    ```
```

Comment:
We could also combine the two create_index methods (scalar and vector).  The inverted index *can* be used in the same way as scalar indices (speeding up filters) but it can also be used for full text search (which is a search and not a filter)

Reply:
I'm going to change the `create_fts_index(column: str)` to `create_fts_index(column: str, index_type="FTS", use_legacy=False)`. We haven't supported use the inverted index as filtering so it still makes sense.

the `create_index` seems has been used for vector index only, that can't be easily changed.